src/content/docs/formatter/index.mdx

```diff
@@ -5,36 +5,44 @@ description: How to use the Biome formatter.
 
 import PackageManagerBiomeCommand from "@/components/PackageManagerBiomeCommand.astro";
 
-Biome is an opinionated formatter.
+Biome is an opinionated formatter that [supports multiple languages](/internals/language-support).
 It follows a similar [philosophy to Prettier](https://prettier.io/docs/en/option-philosophy.html),
 only supporting a few options to avoid debates over styles, turning into debates over Biome options.
 It deliberately [resists the urge to add new options](https://github.com/prettier/prettier/issues/40) to prevent [bike-shed discussions](https://en.wikipedia.org/wiki/Law_of_triviality) in teams so they can focus on what really matters instead.
 
-## Use the formatter with the CLI
 
-By default, the formatter **checks** the code and emit diagnostics if there are changes in formatting:
+## CLI
+
+The following command checks the formatting of the files in the `src` directory.
+It emits text differences if it finds code that is not formatted.
 
 <PackageManagerBiomeCommand command="format ./src" />
 
 If you want to **apply** the new formatting, pass the `--write` option:
 
 <PackageManagerBiomeCommand command="format --write ./src" />
 
-Use the  `--help` flag to know what are the available options:
+The command accept a list of files and diretcories.
+Note that if you pass a glob as a parameter, it will be expanded by your shell.
+The result of the expansion depends on your shell and its configuration.
+For example, some shells don't support the recursive glob `**` or the alternation `{}` in the following command:
```

Comment:
```suggestion
The command accepts a list of files and directories.
Note that if you pass a glob as a parameter, your shell will expand it.
The result of the expansion depends on your shell and its configuration.
For example, some shells don't support the recursive glob `**` or the alternation `{}` in the following command:
```

Maybe we should put this paragraph inside a `:::note`, it's something require a bit of attention

Reply:
Also, we should add a `:::caution` that says that shell expansion of patterns might slow down the command by a lot.